docs/references/service-providers/public-en.md

```diff
@@ -25,7 +25,7 @@ Since we cannot guarantee uptime and stability of the endpoints, do not use them
 |---|---|---|---|
 |[Kaia Foundation](https://www.klaytn.foundation)|`https://public-en-cypress.klaytn.net`|klay,eth,net|Full|
 ||`https://archive-en.cypress.klaytn.net`|klay,eth,net|Archive|
-|[All That Node](www.allthatnode.com)|`https://klaytn-mainnet-rpc.allthatnode.com:8551`|klay,eth,net|Full|
+|[All That Node](www.allthatnode.com)|`https://klaytn-mainnet.g.allthatnode.com/full/evm`|klay,eth,net|Full|
```

Comment:
Thanks @jiseongnoh for the update.
Can you also change testnet to: `https://klaytn-baobab.g.allthatnode.com/full/evm`?

Reply:
Thank you. I updated it.